RecoBTag/DeepFlavour/python/DeepFlavourExporter_cfi.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
```

Comment:
please keep the file name in sync with the instance of the module defined in it:
pfDeepFlavourExporter_cfi.py

Reply:
plugin removed

Reply:
removed with https://github.com/cms-sw/cmssw/pull/19893/commits/de7e130c318f4dc84a506c2e2aef7d32900013a8